src/App.tsx

```diff
@@ -1,61 +1,119 @@
-        </div>
-
-        <button type="submit" data-cy="submitButton">
-          Add
-        </button>
-      </form>
-
-      <section className="TodoList">
-        <article data-id="1" className="TodoInfo TodoInfo--completed">
-          <h2 className="TodoInfo__title">
-            delectus aut autem
-          </h2>
-
-          <a className="UserInfo" href="mailto:Sincere@april.biz">
-            Leanne Graham
-          </a>
-        </article>
-
-        <article data-id="15" className="TodoInfo TodoInfo--completed">
-          <h2 className="TodoInfo__title">delectus aut autem</h2>
-
-          <a className="UserInfo" href="mailto:Sincere@april.biz">
-            Leanne Graham
-          </a>
-        </article>
-
-        <article data-id="2" className="TodoInfo">
-          <h2 className="TodoInfo__title">
-            quis ut nam facilis et officia qui
-          </h2>
-
-          <a className="UserInfo" href="mailto:Julianne.OConner@kory.org">
-            Patricia Lebsack
-          </a>
-        </article>
-      </section>
+    <div className={`App App--${theme}`}>
+      <Header
+        changeTheme={() => changeTheme(theme, setTheme)}
+        theme={theme}
+      />
+
+      <div className={`App__form App__form--${theme}`}>
+        <h1>Add todo form</h1>
+        <form
+          action="/api/todos"
+          method="POST"
+          onSubmit={(event) => handleOnSubmit(
+            event,
+            setTodos,
+            todos,
+            titleOfNewTodo,
+            selectedUserId,
+            usersFromServer,
+            setSelectedUserId,
+            setTitleOfNewTodo,
+            setShowErrorEmptyTitle,
+            setShowErrorUserIsNotChoosen,
+          )}
+        >
+          <div className={`field field--${theme}`}>
+            <input
+              type="text"
+              data-cy="titleInput"
+              onChange={(event) => handleChangeInput(
+                event,
+                setTitleOfNewTodo,
+                setShowErrorEmptyTitle,
+              )}
+              value={titleOfNewTodo}
+              placeholder="Enter a title of your new ToDo"
+            />
+            <br />
+            {showErrorEmptyTitle && (
+              <span className="error">Please enter a title</span>
+            )}
+          </div>
+          <div className={`field field--${theme}`}>
+            <select
+              data-cy="userSelect"
+              value={selectedUserId}
+              onChange={(event) => handleChooseUser(
+                event,
+                setSelectedUserId,
+                setShowErrorUserIsNotChoosen,
+              )}
+            >
+              <option
+                value="0"
+                disabled
+              >
+                Choose a user
+              </option>
+              {usersFromServer.map(user => {
+                return <option value={user.id}>{user.name}</option>;
+              })}
```

Comment:
Don't forget to add the key attribute, also you can use destructuring for `user`